### PR TITLE
#158186607 Move all scripts into a single directory

### PR DESCRIPTION
**Pull request by ayodelevm**
#### What Does This PR Do?
- Creates a bin directory
- Moves all bash scripts into a single directory
- Updates README file

#### Description Of Task To Be Completed
This PR moves all bash scripts into a bin directory in order to make the code base more organised

#### How can this be manually tested?
This can be tested by following the updated instructions in the README

#### What are the relevant Pivotal Tracker stories?
#158186607

### CODE REVIEW

**Comment by adesege** [comment]
Very well done on this PR @ayodelevm. But is this not similar to #1 by @parseendavid?

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ This scripts automatically sets up the project automatically and starts the appl
 ### STARTING THE APP
 Subsequently, if you need to start the application after the initial setup has completed you can run the command:
 ```
-./start.sh
+bin/start.sh
 ```
```
Comment (andela-sjames) [inline README.md]: All shell scripts should be in a single directory called scripts `Acceptance criteria`. 
Did you go through this on `Pivotal Tracker?` @ayodelevm

Comment (andela-sjames) [reply README.md]: Also, you didn't tag anyone on this PR, I will assume it's not ready... 
Do test everything out though

Comment (ayodelevm) [reply README.md]: @andela-sjames didn't take cognisance of that, used bin because I felt it's more popular to use bin... will fix that... Thanks...

**Comment by ayodelevm** [comment]
@adesege separate teams

File: README.md (hunk 1 above)
Comment (ayodelevm) [reply README.md]: @parseendavid I think it will be better to raise this concern on the slack channel, so that people can get involved

Comment (ayodelevm) [reply README.md]: @andela-sjames PR fixed

**Comment by adesege** [comment]
Ok, @ayodelevm. I would like to suggest that a label with the name of the team should be added to each PR for easy indexing and separation of concerns

**Comment by andela-sjames** [comment]
@ayodelevm has this been tested thoroughly?

**Comment by ayodelevm** [comment]
@andela-sjames, yes it has

**Comment by andela-sjames** [comment]
Closing this PR in favor of `FT-setup-docker-#158187265`

**Pull request closed without merging**